docs/openapi/components/schemas/common/HouseBillOfLadingCertificate.yml

```diff
@@ -0,0 +1,80 @@
+$linkedData:
+  term: HouseBillOfLadingCertificate
+  '@id': https://w3id.org/traceability#HouseBillOfLadingCertificate
+title: House Bill of Lading Certificate
+description: >- 
+  A bill of lading issued by a freight forwarder. Often covers a consignment of parcels from various shippers that has been grouped or consolidated by the forwarder. The forwarder may, for example, receive a single groupage bill of lading from the carrier, then issue a series of House B/Ls to the respective shipper.
+  (source: Olegario Llamazares: DICTIONARY OF INTERNATIONAL TRADE, Key definitions of 2000 trade terms and acronyms).
+type: object
+properties:
+  '@context':
+    type: array
+    const:
+      - 'https://www.w3.org/2018/credentials/v1'
+      - 'https://w3id.org/traceability/v1'
+  type:
+    type: array
+    const:
+      - VerifiableCredential
+      - HouseBillOfLadingCertificate
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
```

Comment:
is this supposed to be linked to a Bill Of Lading?

Reply:
No, `Consignment` (as in the example). I'll add the type which I continue with these schemas.